ntbs-service/wwwroot/source/Components/ValidateContactTracing.ts

```diff
@@ -58,9 +58,10 @@ const ValidateContactTracing = Vue.extend({
 
         ValidateModel: function (contactTracingVariables: ContactTracingVariables) {
             let requestConfig = {
-                url: `${this.$props.model}/Validate${this.$props.model}?key=${this.$props.property}`,
+                method: "post" as Method,

```

Comment:
Why is this one different from the one above?

Reply:
The one above was using the wrong enum, have fixed now